dmci/api/worker.py

```diff
@@ -171,6 +175,61 @@ def _check_information_content(self, data):
 
         return valid, msg
 
+    def _add_landing_page(self, data, catalog_url):
+        """Inserts the landing page info in the data bytes string and returns the modified string
+        <related_information>
+           <type>Dataset landing page</type>
+           <resource>https://data.met.no/dataset/{uuid}</resource>
+        </related_information>"""
+
+        if not isinstance(data, bytes):
+            # return False, "Input must be bytes type"
+            logger.critical("Input must be bytes type")
+            sys.exit(1)
+
+        # extract uuid
+        uuid = ""
```

Comment:
self._file_metadata_id (the UUID in the file) can be used since _check_information_content has been run at this point,